lib/linter-plus.coffee

```diff
@@ -110,6 +112,6 @@ class Linter
     @views.destroy()
     @linters.deactivate()
     @commands.destroy()
-    @messages.destroy()
+    @messages.deactivate()
```

Comment:
then as we build them we can just register them into a composite disposable and just dispose it.